docker.sh

```diff
@@ -17,12 +19,16 @@ Copy example-app.conf and fill in blanks."
 
 docker rm -f $CONTAINER_NAME || echo "$CONTAINER_NAME is not yet running."
 
-docker build --tag $IMAGE_NAME $CONTEXT
+start docker_build
+docker build --build-arg COMMONS_BRANCH --tag $IMAGE_NAME $CONTEXT
```

Comment:
I believe there is a docker command syntax feature that pulls the value from the environment if it is not specified, and the change did indeed work- but since the whole mechanism of the COMMONS_BRANCH environment variable has now been ripped out the issue has become irrelevant.